packages/webapp/public/locales/ml/translation.json

```diff
@@ -533,209 +533,209 @@
     "DELETE": {
-      "CONFIRM": "",
-      "HEADER": "",
-      "LINK": "",
-      "MESSAGE": ""
+      "CONFIRM": "ഇല്ലാതാക്കൽ സ്ഥിരീകരിക്കുക",
+      "HEADER": "ചെലവ് ഇല്ലാതാക്കണോ?",
+      "LINK": "ചെലവ് ഇല്ലാതാക്കുക",
+      "MESSAGE": "ഇത് നിങ്ങളുടെ ഫാമിൻ്റെ രേഖകളിൽ നിന്ന് ചെലവ് നീക്കം ചെയ്യും. അത് ഇനി ഒരു സാമ്പത്തിക സംഗ്രഹത്തിലും പ്രതിഫലിക്കില്ല."
     },
-    "DETAILED_HISTORY": "",
+    "DETAILED_HISTORY": "വിശദമായ ചരിത്രം",
     "EDIT_EXPENSE": {
-      "RETIRE_EXPENSE_MESSAGE": "",
-      "RETIRE_EXPENSE_TYPE": "",
-      "RETIRED": "",
-      "TITLE": "",
-      "TITLE_1": "",
-      "TITLE_2": ""
+      "RETIRE_EXPENSE_MESSAGE": "ഈ ചെലവ് തരത്തിൽ നിന്ന് വിരമിക്കുന്നത് ഭാവിയിലെ ചെലവുകൾക്കുള്ള സാധ്യതയുള്ള തിരഞ്ഞെടുപ്പായി ഇത് നീക്കം ചെയ്യും. ഫിനാൻസ് ടാബിൽ ഈ ചെലവ് തരത്തിൻ്റെ ചരിത്രപരമായ സംഭവങ്ങൾ നിങ്ങൾക്ക് തുടർന്നും തിരയാനും ഫിൽട്ടർ ചെയ്യാനും കഴിയും.",
+      "RETIRE_EXPENSE_TYPE": "വിരമിക്കൽ ചെലവ് തരം",
+      "RETIRED": "(വിരമിച്ചു)",
+      "TITLE": "ചെലവ് തിരുത്തുക",
+      "TITLE_1": "ചെലവ് തിരുത്തുക(2-ൽ 1)",
+      "TITLE_2": "ചെലവ് തിരുത്തുക(2-ൽ 2)"
     },
     "ITEM_NAME": "ഇനത്തിൻ്റെ പേര്",
-    "NO_EXPENSE": "",
-    "NO_EXPENSE_YEAR": "",
+    "NO_EXPENSE": "ചെലവൊന്നും കണ്ടെത്തിയില്ല",
+    "NO_EXPENSE_YEAR": "ഈ വർഷത്തെ ചെലവുകളൊന്നും നിങ്ങൾ രേഖപ്പെടുത്തിയിട്ടില്ല",
     "OTHER_EXPENSES_TITLE": "മറ്റ് ചെലവുകൾ",
-    "SUMMARY": "",
-    "TYPE": "",
+    "SUMMARY": "സംഗ്രഹം",
+    "TYPE": "തരം",
     "VALUE": "മൂല്യം"
   },
   "EXPIRED_TOKEN": {
-    "RESET_PASSWORD": "",
-    "RESET_PASSWORD_LINK": ""
+    "RESET_PASSWORD": "ഈ ലിങ്ക് കാലഹരണപ്പെട്ടു.",
+    "RESET_PASSWORD_LINK": "പുതിയ പാസ്‌വേഡ് ലിങ്ക് അയയ്ക്കുക."
   },
   "FARM_MAP": {
     "AREA_DETAILS": {
-      "NETWORK": "",
+      "NETWORK": "നെറ്റ്‌വർക്ക് കണക്റ്റിവിറ്റി പ്രശ്നങ്ങൾ.",
       "PERIMETER": "ചുറ്റളവ്",
       "TOTAL_AREA": "മൊത്തം ഏരിയ"
     },
     "BARN": {
-      "ANIMALS": "",
-      "COLD_STORAGE": "",
-      "EDIT_TITLE": "",
+      "ANIMALS": "ഈ പ്രദേശം മൃഗങ്ങളെ പാർപ്പിക്കാൻ ഉപയോഗിക്കുന്നുണ്ടോ?",
+      "COLD_STORAGE": "ഈ കളപ്പുരയിൽ തണുത്ത സംഭരണം ഉണ്ടോ?",
+      "EDIT_TITLE": "കളപുരയുടെ വിവരങ്ങൾ തിരുത്തുക",
       "NAME": "കളപ്പുരയുടെ പേര്",
-      "TITLE": "",
-      "WASH_PACK": ""
+      "TITLE": "കളപുരയുടെ വിവരങ്ങൾ ചേർക്കുക",
+      "WASH_PACK": "ഈ കളപ്പുരയിൽ വാഷ് ആൻഡ് പാക്ക് സ്റ്റേഷൻ ഉണ്ടോ?"
     },
     "BUFFER_ZONE": {
-      "EDIT_TITLE": "",
+      "EDIT_TITLE": "ബഫർ സോണിൽ ( ജൈവ കൃഷിയെ മറ്റുള്ളവയുടെ കൂടെ കലരാതെ സംരക്ഷിക്കുന്ന മേഖല) തിരുതൽ വരുത്തുക",
       "NAME": "ബഫർ സോണിൻ്റെ പേര്",
-      "TITLE": "",
-      "WIDTH": ""
+      "TITLE": "ബഫർ സോൺ ചേർക്കുക",
+      "WIDTH": "ബഫർ സോണിന്റെ വീതി"
     },
     "BULK_UPLOAD_SENSORS": {
       "DOWNLOAD_FILE": {
-        "DEFAULT": "",
-        "PARTIAL_SUCCESS_BOTTOM_TEXT": "",
-        "PARTIAL_SUCCESS_TOP_TEXT": "",
-        "ROW": "",
-        "SOME_ERRORS": ""
+        "DEFAULT": "നിങ്ങൾ ചെയ്തതിൽ എന്തെങ്കിലും തെറ്റ് സംഭവിച്ചോ?\nസഹായത്തിന് support@litefarm.org എന്നതിൽ ബന്ധപ്പെടുക.",
```

Comment:
And this one I don't see in english

Reply:
I saw that one but said "eh, it makes sense contextually" 😂  It's in a .txt file so no UI to break.